plugin/plugins.qmltypes

```diff
@@ -111,6 +111,7 @@ Module {
         Property { name: "offlineMode"; type: "bool" }
         Property { name: "defaultRoute"; type: "NetworkService"; isReadonly: true; isPointer: true }
         Property { name: "connectedWifi"; type: "NetworkService"; isReadonly: true; isPointer: true }
+        Property { name: "connectedEthernet"; type: "NetworkService"; isReadonly: true; isPointer: true }
```

Comment:
qmlplugindump would put this after "connectingWifi", so I suggest manual edit also puts it there.

Reply:
Done. Never heard about that binary even and on SDK it apparently does not work properly..